src/modules/bricks/volume-bricks/volume-bricks.js

```diff
@@ -64,6 +62,26 @@
             onFilterChange: _filterChange
         };
 
+        vm.volumeDetailConfig = {
+            selectionMatchProp: "fqdn",
+            itemsAvailable: true,
+            showCheckboxes: false
+        };
+
+        vm.volumeDetailColumns = [
+            { header: "Host Name", itemField: "fqdn"},
+            { header: "Brick Path", itemField: "brickPath", htmlTemplate: "/modules/bricks/volume-bricks/brick-path.html" },
+            { header: "Utilization", itemField: "utilization", htmlTemplate: "/modules/bricks/volume-bricks/utilization-path.html" },
+            { header: "Disk Device Path", itemField: "devices", templateFn: function(value, item) { return value[0] } },
+            { header: "Port", itemField: "port" }
+        ];
+
+        vm.actionButtons = [{
+            name: "Dashboard",
+            title: "Dashboard",
+            actionFn: _performAction
```

Comment:
@a2batic What is need of having another function `_performAction` instead of directly calling `_redirectToGrafana` method.

Reply:
Yes, we can use redirectToGrafana directly, I will update the PR.